### PR TITLE
Resolved #2510, Resolved #2517 - BULK: EXP SNAP REVIEW

### LINKED ISSUE #2510: BULK - EXP SNAP REVIEW: new script
Since EXP SNAP identification is so important to counties, I developed a script that will quickly identify cases on a caseload(s) that need to be reviewed for possible EXP SNAP processing. To effectively use this script, this will require agency staff to use the NOTES - EXPEDITED SCREENING script on all SNAP applications (which includes CASH pending cases that do not have SNAP active)

### The script will do the following:
1.	Navigates to REPT/PND1, and gathers the case number, client name, application date and number of days pending. 
2.	Checks each case note header through the application date to see if the NOTES – EXPEDITED SCREENING text is present. 
3.	If the NOTES – EXPEDITED SCREENING case note is not present, or the case note text shows that “client appears expedited” then the case will be added to Excel under the PND1 worksheet.
4.	Steps 2-3 are repeated until all the case numbers have been checked.
5.	An Excel document with a PND1 worksheet will be created, and the PND1 cases that need to be reviewed for potential expedited SNAP criteria. 
6.	Navigates to REPT/PND2, and gathers the case number, client name, application date and number of days pending for cases that are pending for SNAP or CASH. 
7.	Navigates to STAT/PROG to determine if SNAP is pending either as SNAP or MFIP.
8.	For each case that has SNAP pending the script checks each case note header through the application date to see if the NOTES – EXPEDITED SCREENING text is present. 
9.	If the NOTES – EXPEDITED SCREENING case note is not present, or the case note text shows that “client appears expedited” then the case will be added to Excel under the PND2 worksheet.
10.	Steps 7-9 are repeated until all the case numbers have been checked.
11.	An Excel document with a PND1 worksheet will be created, and the PND2 cases that need to be reviewed for potential expedited SNAP criteria. 

Assign to me for the September release.

### LINKED ISSUE #2517: NOTES - EXPEDITED DETERMINATION: case note header updated to read status of EXP determination
The header is static currently. Will be adding EXP determination/action to the header as I will be reading for this header information for the BULK - EXP SNAP REVIEW script (Issue #2510). 

Assign to me for the September release.

### PR DESCRIPTION
**Pull request by IlseFerris**
BLIP for #2510: New script that creates an Excel spreadsheet to quickly identify cases on a caseload(s) that need to be reviewed for possible EXP SNAP processing from REPT/PND1 and REPT/PND2.

BLIP for #2517: Updated case note header to reflect if case is expedited SNAP or not.

### CODE REVIEW

File: Script Files/BULK/BULK - EXP SNAP REVIEW.vbs [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+Const days_pending = 5
+Const appears_exp  = 6      'appears_exp will be carried through to determine if the cases make it to the Excel list or not
+Const case_notes   = 7
+
+For each worker in worker_array
+	back_to_self	'Does this to prevent "ghosting" where the old info shows up on the new screen for some reason
+	Call navigate_to_MAXIS_screen("REPT", "PND1")
+	EMWriteScreen worker, 21, 13
+	transmit
+
```
Comment (CDPotter) [inline Script Files/BULK/BULK - EXP SNAP REVIEW.vbs]: Please add handling for when a worker enters their own X# into the script and they have multiple pages of Pnd1

File: Script Files/BULK/BULK - EXP SNAP REVIEW.vbs [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+Dim PND2_array ()
+ReDim PND2_array (7, 0)
+entry_record = 0
+
+For each worker in worker_array
+	back_to_self	'Does this to prevent "ghosting" where the old info shows up on the new screen for some reason
+	Call navigate_to_MAXIS_screen("REPT", "PND2")
+	EMWriteScreen worker, 21, 13
+	transmit
+
```
Comment (CDPotter) [inline Script Files/BULK/BULK - EXP SNAP REVIEW.vbs]: Please add handling for when a worker enters their own X# into the script and they have multiple pages of Pnd2

**Comment by CDPotter** [comment]
The changes didn't sync again.

**Comment by IlseFerris** [comment]
Try it now, should be good to go

**Pull request merged**